pkg/resource/logging-config/alloy/logging-config.alloy.yaml.template

```diff
@@ -58,3 +61,60 @@ alloy:
           path: /run/log/journal
       - name: alloy-tmp
         emptyDir: {}
+podLogs:
+{{- if .IsWorkloadCluster }}
+- name: namespace-default
+  namespace: giantswarm
+  spec:
+    selector: {}
+    namespaceSelector:
+      matchExpressions:
+      - key: kubernetes.io/metadata.name
+        operator: In
+        values:
+        - kube-system
+        - giantswarm
+- name: pod-selector
+  namespace: giantswarm
+  spec:
+    selector:
+      matchLabels:
+        giantswarm.io/logging: enabled
+    namespaceSelector:
+      matchExpressions:
+      - key: giantswarm.io/logging
+        operator: NotIn
+        values:
+        - enabled
+      - key: kubernetes.io/metadata.name
+        operator: NotIn
+        values:
+        - kube-system
+        - giantswarm
+- name: namespace-selector
+  namespace: giantswarm
+  spec:
+    selector:
+      matchExpressions:
+      - key: giantswarm.io/logging
+        operator: NotIn
+        values:
+        - disabled
+    namespaceSelector:
+      matchExpressions:
+      - key: giantswarm.io/logging
+        operator: In
+        values:
+        - enabled
+      - key: kubernetes.io/metadata.name
+        operator: NotIn
+        values:
+        - kube-system
+        - giantswarm
+{{- else }}
+- name: pod-selector
```

Comment:
We probably want to use the same stuff on MCs for dogfooding so we need to create an issue for this

Reply:
I would like to keep this for later, as this include already a lot of changes I would like to stay as close as possible to the current setup.

Reply:
Yes that's why I said to not forget creating a new issue 😂

Reply:
Issue created: https://github.com/giantswarm/roadmap/issues/3715